src/applications/applications.controller.ts

```
@@ -1,3 +1,15 @@
+import {
+  Body,
+  Controller,
+  Post,
+  Request,
+  UseGuards,
+  ConflictException,
+  Get,
+  Param,
+  ParseIntPipe,
```

ESLint

[error] 'ParseIntPipe' is defined but never used.

(@typescript-eslint/no-unused-vars)